### ISSUE TITLE
About gan loss finetune.

### ISSUE THREAD

**Issue by stalestar** [HyeongminLEE/AdaCoF-pytorch#6]
hello, i see the pretrained model in this repository has not trained with vgg loss and adv loss. in your paper, you finetune the model by λ1 = 0.01; λ = 1; λadv = 0.005, so i have tried to finetune the base model with --loss  '0.01*Charb+0.01*g_Spatial+0.005*g_Occlusion+1.0*VGG+0.005*GAN', but when it undergo the 10st epoch, the model can not run a stable and good performance, what parameters did you set for the finetune training ?

**Comment by stalestar** [comment]
this is my training shell: ‘python train.py --train vimeo_triplet --out_dir out_dir_gan --kernel_size 5 --dilation 1 --gpu_id 0 --batch_size 8 --patch_size 256 --loss '0.01*Charb+0.01*g_Spatial+0.005*g_Occlusion+1.0*VGG+0.005*GAN' --load checkpoint/kernelsize_5/ckpt.pth’

**Comment by HyeongminLEE** [comment]
'GAN' means the vanilla GAN loss for each frame. Our proposed dual-frame adversarial loss is 'FI_GAN'.

Thanks.

**Comment by stalestar** [comment]
Thanks for response. I have try the FI_GAN loss to fineturning the model, it can get a good result in image testsets. But when i run a video,  i fund the motion of objects not being uniform. The edge of motion object will to be really, not be a intermediateness of two neighboring input frames. Would you encounter the problem, and how to solve it ?

**Comment by HyeongminLEE** [comment]
Since GAN Loss is not a completely stable loss (in many other tasks), it could cause some artifacts and could be unstable in the training process. We propose FI_GAN as a possible option to get visually convincing results, but it may not be the best way. We do think that more researches are necessary for VFI + GAN loss.

**Comment by stalestar** [comment]
Yeah, i get. We will try to make some improvement for the work. Thanks for you grate works again, and wish you can get more success in further researching.
